__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../app");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const endpoints= require('../endpoints.json')
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => connection.end());
+
+describe("/api", () => {
+  test("GET - status:200 - returns JSON with available endpoints ", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.endpoints).toEqual(endpoints);
+      });
+    });
+});
+
+describe("/api/categories", () => {
+  test("GET - status:200 - return an array of category objects with slug and description properties ", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.category.length).toBe(4);
+        expect(typeof response.body).toEqual("object");
+        const categories = response.body.category;
+        categories.forEach((category) => {
+          expect(categories).hasOwnProperty("slug");
```

Comment:
Lines 35 and 36 unnecessary as we are asserting against these keys values on lines 37 and 38